### PR TITLE
Vamsi new

### PR DESCRIPTION
**Pull request by Wantrepreneur-10**
Low battery is done and Forgot password

### CODE REVIEW

**Review by seheonm — CHANGES_REQUESTED** [review]
Same as Luke

File: src/KeyPad.java [hunk 1]
```diff
@@ -1,162 +1,183 @@
...
+import javafx.scene.effect.DropShadow;
+import javafx.scene.media.AudioClip;
+import javafx.scene.image.Image;
+import javafx.scene.image.ImageView;
+
+import java.util.Objects;
+
+public class KeyPad extends GridPane {
+
+//    private final Media media = new Media(Objects.requireNonNull(getClass().getResource("audio/KeyPadBeep.mp3")).toString());
```
Comment (mcdougallluke) [inline src/KeyPad.java:19]: The changes you made in the keypad broke the volume control buttons

File: src/SafeGUI.java [hunk 2]
```diff
@@ -81,20 +81,23 @@ public void start(Stage primaryStage) {
         primaryStage.setScene(scene);
         primaryStage.show();
 
-
         try {
```
Comment (mcdougallluke) [inline src/SafeGUI.java:84]: This is causing the program to stay running even after the safe window is closed, can you fix this?

**Review by mcdougallluke — CHANGES_REQUESTED** [review]
Also, what did you implement as the input flow for resetting a user's pin. For instance after 3 incorrect pins, the safe is locked out and requires the master pin, from here, how does the user reset their pin?

**Pull request closed without merging**